### PR TITLE
Basic Interactable Behaviour

### PR DESCRIPTION
**Pull request by OrangePretzel**

### CODE REVIEW

File: Assets/_EscapeRoom/Scripts/UICrosshair.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public class UICrosshair : MonoBehaviour
+    {
+        public enum CrosshairStates
+        {
+            None,
+            Interactable,
+        }
+
+        [SerializeField] private Image _crosshairImage = null;
+        [SerializeField] private Sprite _noneCrosshairSprite = null;
```
Comment (m4rtint) [inline Assets/_EscapeRoom/Scripts/UICrosshair.cs]: is this when theres no crosshair? like its disabled?

File: Assets/_EscapeRoom/Scripts/UICrosshair.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+{
+    public class UICrosshair : MonoBehaviour
+    {
+        public enum CrosshairStates
+        {
+            None,
+            Interactable,
+        }
+
+        [SerializeField] private Image _crosshairImage = null;
```
Comment (m4rtint) [inline Assets/_EscapeRoom/Scripts/UICrosshair.cs]: you can prob do a require componentImage here [RequireComponent(typeof(Image))]

and do GetComponnent<Image>

File: Assets/_EscapeRoom/Scripts/UICrosshair.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/_EscapeRoom/Scripts/UICrosshair.cs]: ah okay, nvm i understand. I think for naming
i would call it nonInteractableCrosshairSprite but i'm okay either way.

Comment (OrangePretzel) [reply Assets/_EscapeRoom/Scripts/UICrosshair.cs]: I added a default crosshair which is just a dot. I renamed this to _default to make it more clear. You can set the image to no sprite which will remove the default crosshair

**Review by m4rtint — APPROVED** [review]

**Pull request merged**